test/Data/PreciseDateTime/Locale.purs

```diff
@@ -0,0 +1,29 @@
+module Test.Data.PreciseDateTime.Locale.Spec where
+
+import Prelude
+
+import Data.BigInt (fromInt)
+import Data.DateTime.Locale (LocalValue(..), Locale(..))
+import Data.Int (toNumber)
+import Data.Maybe (Maybe(..))
+import Data.PreciseDateTime as PDT
+import Data.PreciseDateTime.Locale (fromRFC3339String)
+import Data.RFC3339String (RFC3339String(..))
+import Data.Time.Duration as Dur
+import Data.Time.PreciseDuration (PreciseDuration(..))
+import Test.Data.PreciseDateTime.Spec (dateStringFixture, preciseDateTimeFixture)
+import Test.Spec (Spec, describe, it)
+import Test.Spec.Assertions (shouldEqual)
+
+spec :: forall r. Spec r Unit
+spec =
+  describe "LocalPreciseDateTime" do
+    it "locale" do
```

Comment:
I think this should be `it "fromRFC3339String"`, since that's the function under test.